lib/facter/framework/core/file_loader.rb

```diff
@@ -74,6 +74,7 @@
 require_relative '../../resolvers/networking'
 require_relative '../../resolvers/open_vz'
 require_relative '../../resolvers/os_release'
+require_relative '../../resolvers/os_release_rpm'
```

Comment:
I'm assuming that this new resolver will only be used for Amazon Linux for now, so we can probably just include it in the Amazon-specific stanza on line 238.

Reply:
fixed